vendor.conf

```diff
@@ -54,25 +54,25 @@ github.com/BurntSushi/toml v0.3.1
 
 # kubernetes dependencies
 sigs.k8s.io/yaml v1.1.0
-k8s.io/utils c2654d5206da6b7b6ace12841e8f359bb89b443c
-k8s.io/kubernetes v1.16.0-rc.2
-k8s.io/klog v0.4.0
-k8s.io/cri-api kubernetes-1.16.0-rc.2
-k8s.io/client-go kubernetes-1.16.0-rc.2
-k8s.io/api kubernetes-1.16.0-rc.2
-k8s.io/apiserver kubernetes-1.16.0-rc.2
-k8s.io/apimachinery kubernetes-1.16.0-rc.2
-gopkg.in/yaml.v2 v2.2.2
-gopkg.in/inf.v0 v0.9.0
-golang.org/x/time 85acf8d2951cb2a3bde7632f9ff273ef0379bcbd
+k8s.io/utils e782cd3c129fc98ee807f3c889c0f26eb7c9daf5
+k8s.io/kubernetes v1.16.3
+k8s.io/klog v1.0.0
+k8s.io/cri-api kubernetes-1.16.3
+k8s.io/client-go kubernetes-1.16.3
+k8s.io/api kubernetes-1.16.3
+k8s.io/apiserver kubernetes-1.16.3
+k8s.io/apimachinery kubernetes-1.16.3
+gopkg.in/yaml.v2 v2.2.3
```

Comment:
includes https://github.com/go-yaml/yaml/commit/bb4e33bf68bf89cad44d386192cbed201f35b241 to address CVE-2019-11253